src/libcommon/log/sentry/utility.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Infomaniak kDrive - Desktop
+ * Copyright (C) 2023-2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+#include <string>
+#include "libcommon/utility/types.h"
+#include "libcommon/log/sentry/ptraces.h"
+
+namespace KDC::Sentry {
+static inline std::unique_ptr<AbstractPTrace> SyncSetpToPTrace(SyncStep step, int syncDbId) {
```

Comment:
Uppercase letters are reserved for struct and class names.

Reply:
Did you mean `syncStepToTrace`?

Reply:
Done here: [36c844d](https://github.com/Infomaniak/desktop-kDrive/pull/403/commits/36c844d03aca707b493d8b2b0510e39761a25439)